controllers/storeController.js

```diff
@@ -33,43 +35,69 @@ const getStoreById = async (req, res) => {
 
         if (!store) { 
             return res.status(404).json({
```

Comment:
Ini pake new ApiError ya bang, contohnya dibawah
![image](https://github.com/akbarrahmatm/travel-be-project-team-5/assets/88664674/6e0a48e9-9421-4755-be08-fcff8fc1be52)